src/io/json_reader.cpp

```diff
@@ -91,7 +94,7 @@ struct MyHandler {
         return true;
     }
 
-    DataWriter& w;
+    ed::io::DataWriter& w;
```

Comment:
same question with namespaces.

Reply:
This isn't needed. Though I think it is more clear to use the full NS.